src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Rectangle extends Quadrangle {
+    public Rectangle(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
+        return super.area();
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Figure: rectangle ,");
+        super.draw();
+    }
```

Comment:
The `draw` method should not be printing directly to the console. Consider using a return type that contains the drawing information, which can then be printed elsewhere. This will make your class more flexible and not tied to a single output method.